test/parallel/test-child-process-env.js

```diff
@@ -53,5 +56,8 @@ child.stdout.on('data', function(chunk) {
 
 process.on('exit', function() {
   assert.ok(response.includes('HELLO=WORLD'));
-  assert.ok(response.includes('FOO=BAR'));
+  assert.ok(!response.includes('FOO=BAR'));
```

Comment:
Perhaps simplify this and subsequent absence checks like this?:

```js
  assert.ok(!response.includes('FOO='));
```

In other words, the key should not be included at all no matter what the printed value is. Because this is an absence check, that's more strict than including the value.